From 96acaa87a3689efed6d032521962586abb57f323 Mon Sep 17 00:00:00 2001
From: Rama Chavali <rama.rao@salesforce.com>
Date: Fri, 20 May 2022 19:22:38 +0530
Subject: add log line to indicate envoy discovery request (#39060)

* add log line to indicate envoy discovery request

Signed-off-by: Rama Chavali <rama.rao@salesforce.com>

* add comma

Signed-off-by: Rama Chavali <rama.rao@salesforce.com>
---
 pilot/pkg/xds/ads.go | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/pilot/pkg/xds/ads.go b/pilot/pkg/xds/ads.go
index f6b1781639..56f6fe8ee7 100644
--- a/pilot/pkg/xds/ads.go
+++ b/pilot/pkg/xds/ads.go
@@ -188,6 +188,9 @@ func (s *DiscoveryServer) receive(con *Connection, identities []string) {
 // handles 'push' requests and close - the code will eventually call the 'push' code, and it needs more mutex
 // protection. Original code avoided the mutexes by doing both 'push' and 'process requests' in same thread.
 func (s *DiscoveryServer) processRequest(req *discovery.DiscoveryRequest, con *Connection) error {
+	stype := v3.GetShortType(req.TypeUrl)
+	log.Debugf("ADS:%s: REQUEST %s verson received, %s nonce received %s", stype,
+		con.conID, req.VersionInfo, req.ResponseNonce)
 	if req.TypeUrl == v3.HealthInfoType {
 		s.handleWorkloadHealthcheck(con.proxy, req)
 		return nil
-- 
2.35.3

